docs/index.md

```diff
@@ -39,12 +39,16 @@ GTFS data is now being used by a variety of software applications for many diffe
 ## What is High Quality GTFS?
 
 High quality GTFS is complete, accurate, and up-to-date. This means that it represents how services are currently operating and provides as much information as possible.
+For information on creating high quality data, see the [California Transit Data Guidelines](https://dot.ca.gov/cal-itp/california-transit-data-guidelines), the [GTFS Schedule Best Practices](https://gtfs.org/schedule/best-practices/) and the [GTFS Realtime Best Practices](https://gtfs.org/realtime/best-practices/).
+For evaluating the quality of a dataset, see [Validate](https://gtfs.org/schedule/validate/) page.
 
 ### Complete Data
 
 Quality GTFS includes important service details such as holiday and summer schedule changes, accurate stop locations, and names for routes and headsigns that match other rider-facing materials. Even if an agency works with a vendor to produce GTFS, it’s ultimately up to the agency to ensure that the information presented in print, on board, and online is consistent.
 
-For information on creating high quality data, see the [California Transit Data Guidelines](https://dot.ca.gov/cal-itp/california-transit-data-guidelines) and the [GTFS Best Practices](https://gtfs.org/schedule/best-practices/)
+### Accurate
```

Comment:
Suggest to be "Accurate Data" to maintain consistency with the above "Complete Data"

Reply:
https://github.com/MobilityData/gtfs.org/pull/246/commits/1c0fa0cf476893bdf69ca2c6e0204b21398b03e7